openshift-kube-apiserver/admission/autoscaling/managementcpusoverride/admission.go

```diff
@@ -449,10 +454,13 @@ func podHasBothCPULimitAndRequest(containers []coreapi.Container) bool {
 	return false
 }
 
+// doesNamespaceAllowWorkloadType will return false when a workload type does not match any present ones.
+// When a namespace contains no annotation for workloads we infer that to mean all workload types are allowed.
+// Mutation hook will strip all workload annotation from pods that contain workload annotations in that circumstance.
 func doesNamespaceAllowWorkloadType(annotations map[string]string, workloadType string) bool {
 	v, found := annotations[namespaceAllowedAnnotation]
```

Comment:
The reverse of the condition is a bit confusing. I would at least place the comment right above the `return`

Reply:
Good idea, moved it to the return logic to help with the explanation